corehq/apps/reports/standard/deployments.py

```diff
@@ -165,18 +165,20 @@ def rows(self):
 
         user_id = self.request.GET.get('individual')
         if not user_id:
-            return []
-
-        # security check
-        get_document_or_404(CommCareUser, self.domain, user_id)
-
+            key_args = {}
```

Comment:
Assuming that's right - and that we don't have a couch view we can use, I think the easiest thing is to just force people to select a user.